CMakeLists.txt

```diff
@@ -408,8 +408,10 @@ endif()
 #
 # ZLIB
 #
-set(ZLIB_FIND_REQUIRED ON)
-find_package(ZLIB)
+if (NOT ZLIB_FOUND)
```

Comment:
Nit: Please add a comment here explaining why this is necessary so that one doesn't delete this check by mistake.

Reply:
Done